app/components/views/TransactionsPage/index.js

```diff
@@ -1,41 +1,28 @@
 // @flow
-import React from "react";
-import PropTypes from "prop-types";
-import { autobind } from "core-decorators";
-import TabbedPage from "../../TabbedPage";
-import { FormattedMessage as T } from "react-intl";
-import { withTabSlide } from "../../PageTransitions";
-import transactions from "../../../connectors/transactionsPage";
+import Header from "./Header";
+import { RouteTransition } from "shared";
 
-const propTypes = {
-  router: PropTypes.object.isRequired,
-  isTestNet: PropTypes.bool.isRequired,
-};
+const enterLeft = { atEnter: { offset: -100 }, atActive: { offset: 0 }, atLeave: { offset: 100 }, mapStyles: (styles) => ({ transform: `translateX(${styles.offset}%)` })};
+const enterRight = { atEnter: { offset: 100 }, atActive: { offset: 0 }, atLeave: { offset: -100 }, mapStyles: (styles) => ({ transform: `translateX(${styles.offset}%)` })};
+const opts = { stiffness: 150, damping: 20 };
 
-@autobind
-class Transactions extends React.Component {
-  render() {
-    const { router, isTestNet } = this.props;
-    return (
-      <TabbedPage
-        {...{router}}
-        iconClassName="header-icon-transactions"
-        title={<T id="transactions.title" m="Transactions" />}
-        description={(
-          isTestNet
-            ? <T id="transactions.description.testnet" m="Testnet Decred addresses always begin with letter T and contain 26-35 alphanumeric characters (e.g. TxxXXXXXxXXXxXXXXxxx0XxXXXxxXxXxX0)." />
-            : <T id="transactions.description.mainnet" m="Mainnet Decred addresses always begin with letter D and contain 26-35 alphanumeric characters (e.g. DxxXXXXXxXXXxXXXXxxx0XxXXXxxXxXxX0X)." />
-        )}
-        tabRoutes={[
-          {title: <T id="transactions.tabSend" m="Send" />, route:"/transactions/send"},
-          {title: <T id="transactions.tabReceive" m="Receive" />, route:"/transactions/receive"}
-        ]}
-        children={this.props.children}
-      />
-    );
-  }
-}
+const Transactions = ({ children, location }) => {
+  const tabs = ["send", "receive"];
+  const page = "transactions";
+  const pathname = location.pathname.split("/")[2];
```

Comment:
This is slightly hacky IMO (could break tests?) we should mark it as such and come back around later.

I expect this can be handled better once we upgrade to router v4.  I'm ok with it for now but would like to avoid direct accesses to browser globals as much as we can.

Reply:
Yes.. this will be gone once we upgrade to react router 4

Reply:
Can you had a TODO and create and issue when this PR will be merge just to not forget, please?

Reply:
sure.